parser/shared/src/main/scala/hmda/parser/fi/lar/LarParsingError.scala

```diff
@@ -0,0 +1,4 @@
+package hmda.parser.fi.lar
+
+case class LarParsingError(lineNumber: Int = 0, errorMessages: List[String] = List.empty)
+case class ParsingErrorSummary(transmittalSheetErrors: Seq[List[String]] = Seq.empty, larErrors: Seq[LarParsingError] = Seq.empty)
```

Comment:
I had coded it this way to match the style originally in the `HmdaFileParseState`, but I do agree that it's superfluous.  I'll see if I can trim it down.